package.json

```diff
@@ -13,6 +13,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.3.0",
```

Comment:
Duplicate entry.
Pay attention to [my commit's comment](https://github.com/fakng-agrgtr/fakng-ui/commit/aff0307d3e62a33b602266a236a27b534bb0a76a#r81381299)

Reply:
Done)